### ISSUE TITLE
The "it is not a function" error

### ISSUE THREAD

**Issue by DZakh** [microsoft/playwright-test#270]
I've just installed @playwright/test@1.12.0 and immideatly got an error trying to write following code
```js
const { it, describe } = require("@playwright/test");
```
The error
```
it is not a function
    at Object.<anonymous> (/e2e/tests/smoke.test.js:7:1)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Runner.loadFiles (/e2e/node_modules/folio/out/runner.js:50:17)
    at runTests (/e2e/node_modules/folio/out/cli.js:108:43)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at Command.<anonymous> (/e2e/node_modules/folio/out/cli.js:70:9)
```
The problem is that `console.log(Object.keys(require('@playwright/test')));` logs `[
  '_events',         '_eventsCount',
  '_maxListeners',   '_objects',
  '_connection',     '_type',
  '_guid',           '_parent',
  '_logger',         '_channel',
  '_initializer',    '_forwardPorts',
  'chromium',        'firefox',
  'webkit',          '_android',
  '_electron',       'devices',
  'selectors',       'errors',
  '_selectorsOwner', '_toImpl',
  'expect',          '_baseTest',
  'test',            'default'
]`. There's no `it`.

As I temporary solution I'll stick with `test`.

**Comment by mxschmitt** [comment]
Does using `test` not work for you or was somewhere a broken example?

**Comment by DZakh** [comment]
Well, yeah I can use test, and also it looks like the default export is also the test function. The problem is that all examples in the documentation are importing `it`. For example:
```js
import { it, expect } from "@playwright/test";

it("is a basic test with the page", async ({ page }) => {
  await page.goto("https://playwright.dev/");
  const name = await page.innerText(".navbar__title");
  expect(name).toBe("Playwright");
});
```

**Comment by DZakh** [comment]
But frankly speaking the `test` function also didn't work for me. It said that I can't use `test()` outside of test file, but I've switched to another task so I haven't created an issue.

**Comment by mxschmitt** [comment]
Please refer to the new documentation, this repository is obsolete, we are going to redirect it to the new docs soon: https://playwright.dev/docs/test-intro

**Comment by DZakh** [comment]
Oh, cool! How is it in terms of production readiness now?

**Comment by mxschmitt** [comment]
Its ready!

**Comment by mxschmitt** [comment]
Closing since I fixed the readme in #269
